README.md

```diff
@@ -1 +1,95 @@
 # testcafe-angular-selectors
+
+This plugin provides selector extensions that make it easier to test Angular applications with [TestCafe](https://github.com/DevExpress/testcafe/).
```

Comment:
We should say why it's easier. For example as we done for React. 
`These extensions allow you to select page elements in a way that is native to React.`
I'd like to write a little bit more there. The reason - a person should know what is it and why it's good from the readme.

For example:
This plugin provides selector extensions that make it easier to test Angular applications with TestCafe. These extensions allow to create a Selector to find element on the page by *expressions* that are native for Angular applications (like `binding`, `model` etc.. ).

What do you think @VasilyStrelyaev @miherlosev